client/src/components/Logout/index.js

```diff
@@ -1,22 +1,30 @@
-import React from 'react';
+import React, { Component } from 'react';
 import axios from 'axios';
 
-const Logout = (props) => {
-    axios
+class Logout extends Component {
```

Comment:
I have not done it here, but I think would be good to move all the styles for this component to the separate file inside the folder with the component...
Next time I will try to do it every time I touch the code of any component that does not have a separate styles.css file inside the directory...